compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -1910,6 +1916,9 @@ void t_java_generator::generate_java_struct_parcelable(ostream& out, t_struct* t
         scope_down(out);
       } else {
         switch (bt->get_base()) {
+        case t_base_type::TYPE_I8:
```

Comment:
This probably manifested because I added the default branch (I got a ton of warnings about that on my Linux machine) but I really wonder why this particular missing case has not been detected before by the C++ compiler?